docs/API.md

```diff
@@ -1500,24 +1516,27 @@ stream.close();
 ```
 
 <a name="setObjectRetention"></a>
-### setObjectRetention(String bucketName, String objectName, String versionId, Retention retention, boolean bypassGovernanceRetention)
-`public void setObjectLockRetention(String bucketName, String objectName, String versionId, Retention retention, boolean bypassGovernanceRetention)` _[[Javadoc]](http://minio.github.io/minio-java/io/minio/MinioClient.html#setObjectRetention-java.lang.String-java.lang.String-java.lang.String-io.minio.messages.Retention-boolean-)_
+### setObjectRetention(SetObjectRetentionArgs args)
+`public void setObjectLockRetention(SetObjectRetentionArgs)` _[[Javadoc]](http://minio.github.io/minio-java/io/minio/MinioClient.html#setObjectRetention-io.minio.SetObjectRetentionArgs-)_
 
 Sets retention configuration to an object.
 
  __Parameters__
-| Parameter                     | Type          | Description                     |
-|:------------------------------|:--------------|:--------------------------------|
-| ``bucketName``                | _String_      | Name of the bucket.             |
-| ``objectName``                | _String_      | Object name in the bucket.      |
-| ``versionId``                 | _String_      | Version ID of the object.       |
-| ``config``                    | _[Retention]_ | Object retention configuration. |
-| ``bypassGovernanceRetention`` | _boolean_     | Bypass Governance retention.    |
+ 
+| Parameter        | Type                       | Description                        |
+|:-----------------|:---------------------------|:-----------------------------------|
+| ``args``         | _[SetObjectRetentionArgs]_ | Arguments to set object retention. |
```

Comment:
`Description` needs to be consistent. You could check other APIs e.g. `bucketName | _String_ | Name of the bucket.`.